libtiledbvcf/src/stats/allele_count.h

```diff
@@ -230,6 +230,9 @@ class AlleleCount {
   // Sample names included in the fragment
   inline static std::set<std::string> fragment_sample_names_;
 
+  // Remote flag, true when the VCF dataset is remote.
+  inline static bool remote_ = false;
```

Comment:
Based on previous comments, can we remove the need to track if the array is remote?